warp/yul/main.py

```diff
@@ -45,8 +59,11 @@ def transpile_from_solidity(
             check=True,
             capture_output=True,
         )
+        if result.stderr:
+            print(bcolors.FAIL + bcolors.BOLD + result.stderr.decode("utf-8") + bcolors.ENDC)
+            sys.exit(1)
```

Comment:
We probably don't want to exit, sometimes people print just warnings and stuff to stderr. If the app doesn't choose to fail (no throw), we shouldn't either